index.js

```diff
@@ -127,7 +127,25 @@ class Filru {
           return;
         }
         debug("delete ok", { fullpath });
-        resolve();
+        resolve(fullpath);
+      });
+    });
+  }
+
+  reset() {
```

Comment:
This clears the cache, right? I'd recommend naming it something like:

- purgeAll()
- deleteAllFiles()
- ??